internal/sqlitestore/person.go

```diff
@@ -0,0 +1,72 @@
+package sqlitestore
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/writhe/kudosaurus"
+)
+
+// Person - person model
+type Person struct {
+	gorm.Model
+	SlackID     string
+	Name        string
+	TeamSlackID string
+	Team        Team `gorm:"foreignkey:TeamSlackID;association_foreignkey:SlackID"`
+	IsAdmin     bool
+}
+
+// GetKudos - gets kudos
+func (p Person) GetKudos(db *gorm.DB) []Kudo {
+	var kudos []Kudo
+	db.Where(
+		"owner_id = ? AND is_published = 0",
+		p.ID,
+	).Preload("Author").Preload("Owner").Find(&kudos)
+
+	return kudos
+}
+
+// GetIssuedKudos - gets issued kudos
+func (p Person) GetIssuedKudos(db *gorm.DB) []Kudo {
+	var kudos []Kudo
+	db.Where(
+		"author_id = ? AND is_published = 0",
+		p.ID,
+	).Preload("Author").Preload("Owner").Find(&kudos)
+
+	return kudos
+}
+
+// GetFullData - gets full kudosaurus.Person data from the wrapper
+func (p Person) GetFullData(db *gorm.DB) kudosaurus.Person {
+	var mappedKudos []kudosaurus.Kudo
+	var mappedIssuedKudos []kudosaurus.Kudo
+
+	kudos := p.GetKudos(db)
+	issuedKudos := p.GetIssuedKudos(db)
+	kudosLeft := 10 - len(issuedKudos)
```

Comment:
`10` sounds like something that should be configurable :)

Reply:
Good catch - there is a config variable, but I must've forgotten to use it, here. 🤦